### PR TITLE
Update CLDR version to 41

### PR DESCRIPTION
**Pull request by sdnunca**
This PR is based on #95, updating the CLDR version to 41. 
Additionally, I've updated the list of locales, for both WPCOM and WordPress.org. 

Since version 36, CLDR-JSON was moved to https://github.com/unicode-org/cldr-json, with all data stored in 1 repository. 
I've extracted the WordPress language list from `https://api.wordpress.org/translations/core/1.0/ ` 

### Testing instructions

1. Run `./get-cldr-files.sh`
2. Confirm that a new 41.0.0 folder was created in the `data` folder
3. Run `php prune-cldr-files.php` to delete unneeded files. 
4. Confirm that the size of the `41.0.0` folder is similar to the already existing `29.0.0`

Once the PR is approved, I'll follow up with a PR to also commit the data.

### CODE REVIEW

File: tests/wp-cldr-tests.php [hunk 1]
```diff
@@ -162,15 +165,15 @@ public function test_wpcom_to_cldr_locale_mapping() {
...
+		$wpcom_locales_with_over_1k_translations_at_march_2015 = [ 'az', 'de', 'ja', 'id', 'pt-br', 'it', 'he', 'es', 'fr', 'nl', 'tr', 'hu', 'ru', 'ko', 'ar', 'sq', 'fr-ca', 'fa', 'zh-tw', 'zh-cn', 'sv', 'ga', 'el', 'fi', 'gl', 'bs', 'pt', 'ca', 'sr', 'hr', 'pl', 'nn', 'el-po', 'da', 'gd', 'th', 'cs', 'eo', 'ckb', 'mya', 'cy', 'ro', 'te', 'no', 'bg', 'sk', 'lt', 'ug', 'fr-be', 'vi', 'nb', 'ms', 'km', 'uk', 'eu', 'ky', 'br', 'es-pr', 'fr-ch', 'si', 'ta', 'as', 'mk', 'tl', 'su', 'is', 'et', 'lv', 'af', 'bn', 'oc', 'ur', 'kk', 'ne', 'mwl', 'hi', 'en-gb', 'mhr', 'mn', 'zh', 'sl', 'mr', 'ml', 'so', 'sah', 'fo', 'zh-hk', 'ast' ];
```
[the remark this answers is not in the document]
Comment (sdnunca) [reply tests/wp-cldr-tests.php]: Thank you, I've updated the arrays.

File: tests/wp-cldr-tests.php [hunk 2]
```diff
@@ -162,15 +165,15 @@ public function test_wpcom_to_cldr_locale_mapping() {
...
 		$wp_locales = array_unique( array_merge( $wpcom_locales_with_over_1k_translations_at_march_2015, $wporg_site_languages_menu_at_march_2016 ) );
 
 		// Second, check to see if a CLDR JSON file is available for each one, first excluding known missing
 		// locales, then checking the WP locale is mapped to a CLDR locale, and then checking a language-only
 		// CLDR code. If neither is found, set to `false` so the test fails. Then echo the failed $wp_locale
 		// so we can see in the PHPUnit output which one it was.
-		$known_missing_locales = array( 'su', 'oc', 'mwl', 'mhr', 'azb', 'ceb', 'haz', 'oci' );
+		$known_missing_locales = [ 'su', 'oc', 'mwl', 'mhr', 'azb', 'ceb', 'haz', 'oci' ];
```
[the remark this answers is not in the document]
Comment (sdnunca) [reply tests/wp-cldr-tests.php]: I've updated the list of known missing locales as well.

File: readme.txt [hunk 3]
```diff
@@ -66,6 +72,10 @@ Open up a new issue on GitHub at https://github.com/Automattic/wp-cldr/issues. W
 4. Examples of data available for the locale `hi_IN`, Hindi
 
 == Changelog ==
+= 1.1 (Oct [ ] , 2019) =
```
[the remark this answers is not in the document]
Comment (sdnunca) [reply readme.txt:79]: I've updated the changelog and the plugin version.

[the remark this answers is not in the document]
Comment (dlind1) [reply readme.txt:79]: Was it actually updated for version 36? If not, it'd make more sense to just use the 1.1 tag for the latest version.

[the remark this answers is not in the document]
Comment (sdnunca) [reply readme.txt]: Updated the tag to 1.1, as indeed, version 36 was not actually released.

**Review by dlind1 — APPROVED** [review]
LGTM!

**Pull request merged**